### DISCUSSION TITLE
EPG for Samsung TV Plus?

### DISCUSSION THREAD

**Discussion by CHJ85** [iptv-org/community#200] [Q&A] [answered]
Hi there. Where do I find EPG for the various Samsung TV Plus lists?

**Comment by Sphinxroot** [comment, marked as answer]
hi  personnaly i  use Matt Huismans  (SlyGuy) addons in kodi   EPG is included but you might want to look here https://i.mjh.nz/SamsungTVPlus/ it can be implemented in the playlist , its possbile to add multiple x-tvg-url=  for epg but the loading can be painfull

**Comment by CHJ85** [comment]
That one is pretty good.
However, it doesn't include all the countries.
Like I was hoping for one that has the Netherlands channels.
I should have specified that. Sorry.
But thanks.

**Comment by Sphinxroot** [comment]
sorry for that , maybe Matt Huismans  will add them later , i cant tell  , there 20 country of them ,and now he offer 11 of them

**Comment by CHJ85** [comment]
You mean through the US and UK epgs?
The channel names are the same but the programming is different.
